Src/FluentAssertions/Specialized/DelegateAssertions.cs

```diff
@@ -15,13 +15,19 @@ namespace FluentAssertions.Specialized
     public abstract class DelegateAssertions<TDelegate> : ReferenceTypeAssertions<Delegate, DelegateAssertions<TDelegate>> where TDelegate : Delegate
     {
         private readonly IExtractExceptions extractor;
+        private readonly IClock clock;
 
         protected DelegateAssertions(TDelegate @delegate, IExtractExceptions extractor) : base(@delegate)
         {
```

Comment:
If this constructor is used, `clock` will be `null` in the constructed instance.
Can we call ``: this(...)` as in the other classes?

Reply:
@dennisdoomen and this one